crates/executor/src/executor.rs

```diff
@@ -321,21 +335,27 @@ pub fn execute<DB: StateProvider>(
         revm_wrap::fill_tx_env(&mut evm.env.tx, transaction);
 
         // Execute transaction.
-        let (revm::ExecutionResult { exit_reason, gas_used, logs, gas_refunded, .. }, state) =
-            evm.transact();
+        let out = evm.transact();
 
-        tracing::trace!(target:"evm","Executing transaction {:?}, gas:{gas_used} refund:{gas_refunded}",transaction.hash());
+        //let out = evm.inspect(revm::inspectors::CustomPrintTracer::new());
+        //tracing::trace!(target:"evm","Executing transaction {:?}, \n:{out:?}: {:?}
+        // \nENV:{:?}",transaction.hash(),transaction,evm.env);
```

Comment:
Useful for debugging, in future, this will be behind some `config` flag, maybe we could even specify what Inspector we want to use, as in `CustomPrinter` or `JsonTraces`  etc.